db-dump-entrypoint.sh

```diff
@@ -0,0 +1,53 @@
+#!/usr/bin/env sh
+# Alternative entrypoint for the inmanta orchestrator container.
+# Instead of starting the orchestrator, it starts a cron daemon, to allow
+# pg_dump to periodically run and export a database dump.
+
+set -x
+set -e
+
+# Install cron, logrotate and tini
+apt-get update
+apt-get install -y cron logrotate  tini
+
+# Configure db dump script, save container env vars in script to simplify
+# container configuration
+if [ ! -f /db_dump ]; then
+    cat > /db_dump <<EOF
+#!/usr/bin/env sh
+$(export)
+pg_dump -f /var/lib/postgresql/data/db_dump.sql
```

Comment:
As for all applications, `/var/lib/<application>` is considered owned by that application. So end-users are not supposed to write their own data into it. This might just work fine, but it's considered a bad practice.

Reply:
I didn't know that, cool.  But I think here it is okay because:
1. Here the pg_dump is the application, not the end user (it has its own container, for its own specific task)
2. I did it in the first place because the postgres image will automatically define a volume for that folder.  Which means that if we don't want to "leak" volumes on every start of the container, we should mount a volume ourselves, and having data in the folder seemed like the easiest way to suggest the user to put a volume there.